modules/aws-iam-user/variables.tf

```diff
@@ -33,13 +33,14 @@ variable "pgp_key" {
   default     = ""
 }
 
-variable "create-new-policy" {
-  type        = bool
-  default     = false
-  description = "If value true will create new policy"
-}
 variable "policy" {
   description = "IAM policy resource"
   type        = any
   default     = null
 }
+
+variable "create_policy" {
+  description = "Whether create a policy or not."
+  type        = bool
+  default     = true
```

Comment:
by default policy is null and create_policy is true, seems not logically connected, this two are connected and lets have default values in sync